newton/_src/viewer/viewer_usd.py

```
@@ -182,9 +222,53 @@ def log_mesh(
             pass
 
         # how to hide the prototype mesh but not the instances in USD?
-        # mesh_prim.GetVisibilityAttr().Set("inherited" if not hidden else "invisible", self._frame_index)
+        mesh_prim.GetVisibilityAttr().Set("inherited" if not hidden else "invisible", self._frame_index)
+
+    # log a set of instances as individual mesh prims, slower but makes it easier
+    # to do post-editing of instance materials etc. default for Newton shapes
+    def log_instances(self, name, mesh, xform, scale, color, material):
```

Ruff (0.13.1)

Unused method argument: `material`

(ARG002)